### ISSUE TITLE
Include date and/or version in Bylaws

### ISSUE THREAD

**Issue by hlapp** [OBF/obf-docs#36]
It would be useful, I think, to be able to identify documents floating around somewhere as to which version of the Bylaws they represent. Right now, the only way to identify that is by comparison of the text.

Or perhaps we can start with publishing MD5 checksums?

**Comment by peterjc** [comment]
Version numbers make sense, plus a changes section (at the end of the file?) which ideally we'd back date for the recent revisions (joining SPI and the minor changes at the October 2016 board meeting).

Were you suggesting MD5 (and/or SHA256) checksum of the MarkDown files? That ought to be stable while the HTML/PDF/etc would not be.

**Comment by hlapp** [comment]
> Were you suggesting MD5 (and/or SHA256) checksum of the MarkDown files? That ought to be stable while the HTML/PDF/etc would not be.

What I had in mind was for each of them. Why would HTML and PDF be less stable than the MD (for the purposes of verifying whether one has the exact same file as is posted in the repo)?

**Comment by peterjc** [comment]
Because rebuilding the document is likely to give different output (e.g. version of pandoc changed), but if you mean recording the checksum as committed that could work.

I think versioning is more important than checksums and should be prioritised.

**Comment by nlharris** [comment]
Did this ever get done? Can we just add "Last updated" dates to these docs as a simple solution?

**Comment by peterjc** [comment]
+1 on last date stamp of latest content change (i.e. would not need to be changed for things like rebuilding the HTML/PDF).

I'd be fine with adding that to the current version as a footnote without formal membership approval (just a PR with board approval seems enough).

**Comment by hlapp** [comment]
> I'd be fine with adding that to the current version as a footnote without formal membership approval (just a PR with board approval seems enough).

I agree, this wouldn't be changing the text of the bylaws themselves.
